src/hoc/selectable-enhance.js

```diff
@@ -76,12 +76,20 @@ export const SelectableContainerEnhance = (Component) => {
           key: this._keyIndex,
           style: mergedChildrenStyles,
           nestedItems: child.props.nestedItems.map((child) => this._extendChild(child, styles, selectedItemStyle)),
+          initiallyOpen: child.props.nestedItems.reduce(this._hasSelectedDescendant, false),
```

Comment:
@mbrookes @oliviertassinari @alitaheri 

Sorry, I haven't completely figured out how this component is working in my brief glance at it. I agree that the new behavior is the correct behavior (thanks @mbrookes), but it seems like the changes are taking control of the `initiallyOpen` prop away?  For example, if none of the list items are selected, is there still a way for the `initiallyOpen` prop to open the nested list items?

>  it could take an enum of ['true' | 'false' | 'auto'], where true opens all the items, false is current behaviour (none open), and auto is the new feature?
I think this is a fair compromise, not sure if it's worth the effort. We still are going to have to fix the `valueLink` API on list so we could also reinvest the API at that point and not change props now.